.changeset/quiet-ties-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/storage": patch
+---
+
+Remove ipfs-2.thirdwebcdn.com due to poor domain authority
```

Comment:
let's remove this changeset (no reason to have 2 changesets for the same "change")

Reply:
Done

Reply:
you... kept this one? wasn't the other one better?